src/Confluent.Kafka/IProducer.cs

```diff
@@ -103,5 +67,23 @@ public interface IProducer<TKey, TValue> : IProducerBase, IClient
             TopicPartition topicPartition,
             Message<TKey, TValue> message,
             Action<DeliveryReport<TKey, TValue>> deliveryHandler = null);
+
+        
+        /// <summary>
+        ///     Refer to <see cref="Confluent.Kafka.Producer{TKey,TValue}.Poll(TimeSpan)" />
+        /// </summary>
+        int Poll(TimeSpan timeout);
+
+
+        /// <summary>
+        ///     Refer to <see cref="Confluent.Kafka.Producer{TKey,TValue}.Flush(TimeSpan)" />
+        /// </summary>
+        int Flush(TimeSpan timeout);
+
+
+        /// <summary>
+        ///     Refer to <see cref="Confluent.Kafka.Producer{TKey,TValue}.Flush(CancellationToken)" />
+        /// </summary>
+        void Flush(CancellationToken cancellationToken = default(CancellationToken));
```

Comment:
I thought we were not to include the cancellationToken methods until we could properly implement cancellable librdkafka calls, yesno?